client/signup/controller.js

```diff
@@ -245,6 +245,20 @@ export default {
 				experiment.variationName === 'treatment_scrambled';
 		}
```

Comment:
The experiment 21671-explat-experiment, `calypso_signup_onboarding_site_goals_survey_i2`, is disabled and can probably be removed and cleaned up. cc: @brainiaq